etp-backend/src/main/clj/solita/etp/service/valvonta_oikeellisuus/email.clj

```diff
@@ -7,18 +7,33 @@
             [solita.etp.service.valvonta-oikeellisuus.toimenpide :as toimenpide]
             [solita.common.map :as map]
             [solita.common.time :as time]
-            [solita.common.logic :as logic])
-  (:import (java.time LocalDate)))
+            [solita.common.logic :as logic]
+            [solita.etp.service.valvonta-oikeellisuus.asha :as asha-valvonta-oikeellisuus]
+            [clojure.java.io :as io])
+  (:import (java.time LocalDate)
+           (java.io File)))
 
-(defn- send-email! [{:keys [to subject body]}]
+(defn- send-email! [{:keys [to subject body]} & [{:keys [reply-to-email reply-to-name]}]]
   (smtp/send-text-email! config/smtp-host
                          config/smtp-port
                          config/smtp-username
                          config/smtp-password
                          config/email-from-email
                          config/email-from-name
                          [to]
-                         subject body "html"))
+                         subject body "html"
+                         reply-to-email reply-to-name))
+
+(defn- send-email-with-attachment! [{:keys [to subject body attachment]} & [{:keys [reply-to-email reply-to-name]}]]
```

Comment:
vähän sekavan näköinen rajapinta kun on kaksi objektia

Reply:
ja riittäisikö jos parametrina olisi vain reply? :: boolean

Reply:
eli common/smtp olisi yksi parametri reply? ja tällä kuvataan kumpi reply-to address laitetaan. Tuskin näitä tulee muita kuin reply ja no-reply osoitteet.